packages/tergent/build.sh

```diff
@@ -1,10 +1,10 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/aeolwyr/tergent
+TERMUX_PKG_HOMEPAGE=https://github.com/danog/tergent
 TERMUX_PKG_DESCRIPTION="A cryptoki/PKCS#11 library for Termux that uses Android Keystore as its backend"
 TERMUX_PKG_LICENSE="GPL-3.0"
-TERMUX_PKG_VERSION=1.0.0
+TERMUX_PKG_VERSION=1.0.1
 TERMUX_PKG_REVISION=2
-TERMUX_PKG_SRCURL=https://github.com/aeolwyr/tergent/archive/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=0b59cf0ced3f693fb19396a986326963f3763e6bf65d3e56af0a03d206d69428
+TERMUX_PKG_SRCURL=https://github.com/danog/tergent/archive/${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
Upstream change is only allowed in these cases:
* Ownership officially has been transferred to another person or organization.
* Old upstream is no longer being developed.

So please if you have some changes, put a patch file into directory with build.sh file and revert back changes to HOMEPAGE, URL and VERSION fields.

Reply:
In this case, it really seems like the old upstream isn't being developed anymore, the last commit was in August, and my PR is inactive since September.

Anyway, I suppose I can create a simple patch, no biggie)

Reply:
Add patch now, your fork has a single change (if omit README) - https://github.com/danog/tergent/commit/87ab008deb2bd7701da6e91f3998e51e68ec2bcf.

When you will have something else to add besides "automatic fingerprint unlock", then it can be switched.